frontend/src/app/playback/playback.component.css

```diff
@@ -1,10 +1,13 @@
 /* src/app/playback/playback.component.css */
 
-.container {
+--first-color: #fcf9f4;
```

Comment:
Probably overkill for now, but you can use CSS custom properties (a.k.a. variables) for this. If the same colours are used in various files, you can even move them out to a shared global styles file and import them in to any component that needs them.

I'd also suggest switching to SCSS. The syntax is very similar to vanilla CSS, but you get the advantage of defining nested rules, e.g.

```
.my-button {
  color: red;

  &.bold-text {
    font-weight: bold;
  }

  &:hover {
    color: blue;
  }
}
```

That would also give you access to using the neater `$first-color` syntax for variables (compared to the CSS `var(--first-color)`).